iceoryx_hoofs/vocabulary/include/iox/detail/string.inl

```diff
@@ -238,6 +238,22 @@ inline bool string<Capacity>::unsafe_assign(const char* const str) noexcept
     return true;
 }
 
+template <uint64_t Capacity>
+inline void
+string<Capacity>::unsafe_raw_access(const iox::function<void(char*, const uint64_t, const uint64_t)>& func) noexcept
+{
+    func(m_rawstring, m_rawstringSize, Capacity);
```

Comment:
What do you think about setting `m_rawstrin[Capacity]` to `'\0'` before calling `func`?

Reply:
I do not see any case where having the termination at the capacity size will help to not have troubles.